scripts/gulpfiles/test_tasks.js

```diff
@@ -32,48 +32,84 @@ const BOLD_GREEN = '\x1b[1;32m';
 const BOLD_RED = '\x1b[1;31m';
 const ANSI_RESET = '\x1b[0m';
 
-let failerCount = 0;
+class Tester {
```

Comment:
What's the advantage of encapsulating these in a class?

Reply:
Mainly just to remove the `failerCount` global variable, and allow us to also count successes without adding a second global.